pyproject.toml

```diff
@@ -9,7 +9,7 @@ version = "0.1.0"
 dependencies = [
   'ansible-core~=2.15.9',
   'ansible-anonymizer~=1.5.0',
-  'ansible-risk-insight~=0.2.6',
+  'ansible-risk-insight~=0.2.7',
```

Comment:
side note: You don't really need to change this one `~= 0.2.6` already includes 0.2.7 and having to tight dependency can create dependency hell for the users.

Reply:
Ah, IDK that. Thanks.

Since `0.2.7` broke a test (from `0.2.6`) it's possibly a good thing to explicitly bump it. WDYT?

I can close this PR if you think it is better to do so.

Reply:
No worries, it was a minor observation :-).